trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable
```

Comment:
- [ ] Kindly check your else statement, you should specify the correct name of the nameable is returned.